invesalius/gui/dialogs.py

```diff
@@ -2832,14 +2828,20 @@ def GetConnSelected(self):
 
 
 class PanelFFillThreshold(wx.Panel):
-    def __init__(self, parent, config, ID=-1, style=wx.TAB_TRAVERSAL | wx.NO_BORDER):
+    def __init__(
+        self,
+        parent: wx.Window,
+        config: "FFillSegmentationConfig",
```

Comment:
str

Reply:
I am sorry but I don't understand this part. Can you please point out which variable is `str`?

Reply:
Are you sure that `config` is a `str`? The interface given in the methods exactly matches `FFillSegmentationConfig`.

Reply:
Sorry, my mistake. It's `FFillSegmentationConfig`